whois-query/src/test/java/net/ripe/db/whois/query/integration/TemplateTestIntegration.java

```diff
@@ -378,54 +384,60 @@ hierarchical authorisation. Protects creation of objects directly (one
-
+                                
                 mnt-routes
-
+                                
                    This attribute references a maintainer object which is used in
                    determining authorisation for the creation of route objects.
                    After the reference to the maintainer, an optional list of
                    prefix ranges inside of curly braces or the keyword "ANY" may
                    follow. The default, when no additional set items are
                    specified, is "ANY" or all more specifics. Please refer to
                    RFC-2622 for more information.
-
+                                
                      <mnt-name> [ { list of <address-prefix-range> } | ANY ]
-
+                                
                 mnt-irt
-
+                                
                    May appear in an inetnum or inet6num object. It points to an irt
                    object representing a Computer Security Incident Response Team (CSIRT)
                    that handles security incidents for the address space specified by the
                    inetnum or inet6num object.
-
+                                
                      An irt name is made up of letters, digits, the character
                      underscore "_", and the character hyphen "-"; it must start
                      with "irt-", and the last character of a name must be a
                      letter or a digit.
-
+                                
                 created
-
+                                
                    This attributes reflects when the object was created in
                    ISO8601 format (yyyy-MM-dd'T'HH:mm:ssZ).
-
+                                
                      Attribute generated by server.
-
+                                
                 last-modified
-
+                                
                    This attributes reflects when the object was last changed in
                    ISO8601 format (yyyy-MM-dd'T'HH:mm:ssZ).
-
+                                
                      Attribute generated by server.
-
+                                
                 source
-
+                                
                    Specifies the registry where the object is registered. Should be
                    "RIPE" for the RIPE Database.
-
+                                
                      Made up of letters, digits, the character underscore "_",
                      and the character hyphen "-"; the first character of a
                      registry name must be a letter, and the last character of a
-                     registry name must be a letter or a digit."""));
+                     registry name must be a letter or a digit.
+                                
+                                
+                % This query was served by the RIPE Database Query Service version 0.1-TEST (MHERRAN-PRO)
```

Comment:
This final comment uses your laptop name, can you remove " (MHERRAN-PRO)" from the comparison?